### ISSUE TITLE
flutter_math_fork-0.7.1 does not support flutter sdk: '>=3.2.0-191.0.dev <4.0.0'

### ISSUE THREAD

**Issue by The-Redhat** [simpleclub-extended/flutter_math_fork#85]
**Describe the bug**
The package does not compile when used in the flutter beta channel. 

**To Reproduce**
1. Switch to the flutter beta channel
2. Try to run the app

**Additional context**
```
Error (Xcode): ../../.pub-cache/hosted/pub.dev/flutter_math_fork-0.7.1/lib/src/widgets/selection/gesture_detector_builder_selectable.dart:38:22: Error: Type 'TapDragUpDetails' not found.
```

**Comment by manoletre** [comment]
It is not compiling for me on flutter Channel Stable 3.16.0.

Here is my flutter doctor:

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.16.0, on macOS 13.4 22F66 darwin-arm64, locale en-GB)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 14.3.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.3)
[✓] VS Code (version 1.84.2)
[✓] Connected device (2 available)
[✓] Network resources
```

It throws this error:

```
: Error: Type 'TapDragUpDetails' not found.
gesture_detector_builder_selectable.dart:38
  void onSingleTapUp(TapDragUpDetails details) {
                     ^^^^^^^^^^^^^^^^

: Error: 'TapDragUpDetails' isn't a type.
gesture_detector_builder_selectable.dart:38
  void onSingleTapUp(TapDragUpDetails details) {

                     ^^^^^^^^^^^^^^^^
Failed to compile application.
```

**Comment by michalsrutek** [comment]
This should be fixed in https://github.com/simpleclub-extended/flutter_math_fork/pull/87

**Comment by Sese-Schneider** [comment]
Fixed #87
